functions/src/transact.ts

```diff
@@ -69,7 +69,7 @@ export const transact = functions
 
       const newTxnDoc = firestore.collection(`txns/`).doc()
 
-      const txn: Txn = removeUndefinedProps({
+      const txn = removeUndefinedProps({
```

Comment:
I took the type away because the type is wrong. This doc is not known to be of type `Txn` because we don't validate the input data for this cloud function to ensure that it is.